client/src/App.tsx

```diff
@@ -1,37 +1,77 @@
 import "./App.scss";
-import PlayerScore from "./components/PlayerScore/PlayerScore";
-import { DEFAULT_ROUNDS, DEFAULT_PLAYERS } from "./constants/constants";
-import { Header } from "./components/header/Header";
-import { initialiseGame } from "./service/game";
 import { useState } from "react";
+import { Header } from "./components/Header/Header";
+import PlayerScore from "./components/PlayerScore/PlayerScore";
 import { Game } from "./types/game/game.types";
+import { DEFAULT_ROUNDS, DEFAULT_PLAYERS } from "./constants/constants";
+import { initialiseGame, getInitialPlayerNames } from "./service/game";
+import validatePlayerName from "./validation/validate_player_name";
+import { PlayerNameAndId } from "./types/player/player.types";
+
 function App() {
   const [game, setGame] = useState<Game>(
     initialiseGame(DEFAULT_PLAYERS, DEFAULT_ROUNDS)
   );
   const player1 = game.players[0];
   const player2 = game.players[1];
 
+  const [inputPlayerNames, setInputPlayerNames] = useState<PlayerNameAndId[]>(
+    getInitialPlayerNames(game.players)
+  );
+
+  const updatePlayerName = (id: string, value: string) => {
```

Comment:
I wasn't expecting this code in this PR. Maybe you want to do a compare of this branch against https://github.com/TeamTopTrumps/top-trumps-game/pull/26 ?

Reply:
I think it's because it branches off the update-player-score branch? Not off main? So that one needs to be merged first?

Reply:
yeah you can also set your PR to be reviewed off another branch (other than main) so we can see the smaller set of changes. 
I'll look over #26 shortly anyway

Reply:
Ah sorry I didn't know that! Is there a way for me to change it now? Or should I close this PR and try and open a new one to be reviewed off update-player-score?